lib/strip_attributes/matchers.rb

```diff
@@ -6,26 +6,35 @@ module Matchers
     # RSpec Examples:
     #
     #   it { is_expected.to strip_attribute(:first_name) }
+    #   it { is_expected.to strip_attributes(:first_name, :last_name) }
     #   it { is_expected.not_to strip_attribute(:password) }
+    #   it { is_expected.not_to strip_attributes(:password, :encrypted_password) }
     #
     # Minitest Matchers Examples:
     #
     #   must { strip_attribute :first_name }
+    #   must { strip_attributes(:first_name, :last_name) }
     #   wont { strip_attribute :password }
-    def strip_attribute(attribute)
-      StripAttributeMatcher.new(attribute)
+    #   wont { strip_attributes(:password, :encrypted_password) }
+    def strip_attribute(*attributes)
+      StripAttributeMatcher.new(*attributes)
     end
 
+    alias_method :strip_attributes, :strip_attribute
+
     class StripAttributeMatcher
-      def initialize(attribute)
-        @attribute = attribute
+      def initialize(*attributes)
+        @attributes = attributes
         @options = {}
       end
 
       def matches?(subject)
-        subject.send("#{@attribute}=", " string ")
-        subject.valid?
-        subject.send(@attribute) == "string" and collapse_spaces?(subject)
+        @attributes.map do |attribute|
+          @attribute = attribute
+          subject.send("#{@attribute}=", " string ")
+          subject.valid?
+          subject.send(@attribute) == "string" and collapse_spaces?(subject)
+        end.all?
```

Comment:
[PLEASE CHANGE] no need to pass over the array twice:

```ruby
        @attributes.all? do |attribute|
          @attribute = attribute
          subject.send("#{@attribute}=", " string ")
          subject.valid?
          subject.send(@attribute) == "string" and collapse_spaces?(subject)
        end
```

Reply:
Good catch! Updated, and I also removed the extra unnecessary splats (`*`)

Reply:
Would you like me to squash the commits into one, or is there an automatic way github can do that with the merge (I'm used to gitlab)?